src/Microsoft.SqlTools.ServiceLayer/QueryExecution/DataStorage/StorageDataReader.cs

```diff
@@ -98,7 +98,7 @@ public Task<bool> ReadAsync(CancellationToken cancellationToken)
         /// <returns>The value of the given column</returns>
         public object GetValue(int i)
         {
-            return sqlDataReader == null ? DbDataReader.GetValue(i) : sqlDataReader.GetValue(i);
+            return sqlDataReader == null ? DbDataReader.GetValue(i) : sqlDataReader.GetSqlValue(i);
```

Comment:
I check that both in product and test code the data reader is already SqlDataReader, @kburtram  do you remember why we need to check whether is a SqlDataReader?

Reply:
no, I don't recall the details on why we're checking this here.